app/handler/commands.js

```diff
@@ -803,6 +804,39 @@ exports.handleMessage = function (steamID, message) {
             }
             client.chatMessage(steamID, 'Price check has been requested, the item will be checked.');
         });
+    } else if (isAdmin && command === 'name') {
+        // This has already been used but since I'm planning on rewriting this for user extensions it will remain.
+        const newName = removeCommandFromMessage(message, command);
+
+        community.editProfile({
+            name: newName
+        }, function (err) {
+            if (err) {
+                const errorMessage = 'Error while changing name: ' + err.message;
+
+                log.warn(errorMessage);
+                client.chatMessage(steamID, errorMessage);
+
```

Comment:
Over complicating something as simple as logging the error and sending a message back.

```js
log.warn('Error updating name: ', err);
client.chatMessage(steamID, 'Error updating name: ' + err.message);
```